arwen/cryptoapi/cryptoei.go

```diff
@@ -340,6 +425,32 @@ func v1_4_ripemd160(context unsafe.Pointer, dataOffset int32, length int32, resu
 	return 0
 }
 
+//export v1_4_managedRipemd160
+func v1_4_managedRipemd160(context unsafe.Pointer, inputHandle int32, outputHandle int32) int32 {
+	runtime := arwen.GetRuntimeContext(context)
+	managedType := arwen.GetManagedTypesContext(context)
+	crypto := arwen.GetCryptoContext(context)
+	metering := arwen.GetMeteringContext(context)
```

Comment:
Sometimes `StartGasTracing` is used, while other times `UseGasAndAddTracedGas` is used. Is this all right?

Reply:
yes. both can be used